data/mods/Aftershock/monsterdrops/monsterdrops.json

```diff
@@ -91,5 +91,214 @@
+    "type": "harvest",
+    "entries": [
+      { "drop": "alien_flesh", "type": "flesh", "mass_ratio": 0.3 },
+      { "drop": "alien_meat_scrap", "type": "flesh", "mass_ratio": 0.03 },
+      { "drop": "alien_lung", "type": "flesh", "mass_ratio": 0.0035 },
+      { "drop": "alien_liver", "type": "offal", "mass_ratio": 0.01 },
+      { "drop": "alien_brain", "type": "flesh", "mass_ratio": 0.005 },
+      { "drop": "alien_sweetbread", "type": "flesh", "mass_ratio": 0.002 },
+      { "drop": "alien_kidney", "type": "offal", "mass_ratio": 0.002 },
+      { "drop": "alien_stomach", "scale_num": [ 1, 1 ], "max": 1, "type": "offal" },
+      { "drop": "alien_bone", "type": "bone", "mass_ratio": 0.15 },
+      { "drop": "sinew", "type": "bone", "mass_ratio": 0.00035 },
+      { "drop": "raw_alien_leather", "type": "skin", "mass_ratio": 0.02 },
+      { "drop": "alien_fat", "type": "flesh", "mass_ratio": 0.07 },
+      { "drop": "alien_organs", "type": "offal", "mass_ratio": 0.01 }
+    ]
+  },
+  {
+    "id": "alien_large_leather",
+    "//": "drops large stomach",
+    "type": "harvest",
+    "entries": [
+      { "drop": "alien_flesh", "type": "flesh", "mass_ratio": 0.32 },
+      { "drop": "alien_meat_scrap", "type": "flesh", "mass_ratio": 0.01 },
+      { "drop": "alien_lung", "type": "flesh", "mass_ratio": 0.0035 },
+      { "drop": "alien_liver", "type": "offal", "mass_ratio": 0.01 },
+      { "drop": "alien_brain", "type": "flesh", "mass_ratio": 0.005 },
+      { "drop": "alien_sweetbread", "type": "flesh", "mass_ratio": 0.002 },
+      { "drop": "alien_kidney", "type": "offal", "mass_ratio": 0.002 },
+      { "drop": "alien_stomach_large", "scale_num": [ 1, 1 ], "max": 1, "type": "offal" },
+      { "drop": "alien_bone", "type": "bone", "mass_ratio": 0.15 },
+      { "drop": "sinew", "type": "bone", "mass_ratio": 0.00035 },
+      { "drop": "raw_alien_leather", "type": "skin", "mass_ratio": 0.02 },
+      { "drop": "alien_fat", "type": "flesh", "mass_ratio": 0.07 },
+      { "drop": "alien_organs", "type": "offal", "mass_ratio": 0.01 }
+    ]
+  },
+  {
+    "id": "alien_small_wool",
+    "//": "does not drop stomach or other small organs",
+    "type": "harvest",
+    "entries": [
+      { "drop": "alien_flesh", "type": "flesh", "mass_ratio": 0.28 },
+      { "drop": "alien_meat_scrap", "type": "flesh", "mass_ratio": 0.05 },
+      { "drop": "alien_lung", "type": "flesh", "mass_ratio": 0.0035 },
+      { "drop": "alien_liver", "type": "offal", "mass_ratio": 0.01 },
+      { "drop": "alien_brain", "type": "flesh", "mass_ratio": 0.005 },
+      { "drop": "alien_bone", "type": "bone", "mass_ratio": 0.15 },
+      { "drop": "sinew", "type": "bone", "mass_ratio": 0.00035 },
+      { "drop": "wool_staple", "type": "skin", "mass_ratio": 0.02 },
+      { "drop": "alien_fat", "type": "flesh", "mass_ratio": 0.07 },
+      { "drop": "alien_organs", "type": "offal", "mass_ratio": 0.01 }
+    ]
+  },
+  {
+    "id": "alien_large_wool",
```

Comment:
Will animals from the wool body type be able to be shaved like sheep can be in the pet menu?  I can't remember if sheep can be sheared or if that never got added. Just something to think of.

Reply:
Probably? I do think sheep can be shaved, yes